test/composition/default-declarative-shadow-dom.html

```diff
@@ -99,15 +99,15 @@ <h2>Custom Shadow DOM alternative composition</h2>
                     .sameHTMLString_ignoringShadyCSSPolyfillClasses(REFERENCE_ALTERNATIVE_COMPOSITION);
             });
         });
-        describe('after composition is changed from custom subset to explicit', function() {
+        describe('after composition is changed from custom subset to temporary', function() {
```

Comment:
AS before maybe, I'd write it more like: "after composition is changed usin `updateComposition` from custom subset to temporary" because that's what is actually tested not ,changing it using `scInclude.shadowRoot`